Revise rules for the use of enable and disable (#2701)
opensearch-project/documentation-website: TERMS.md

**disable**

> Use *disable* to describe making a feature or command unavailable. Examples:

Reviewer: ```suggestion
Use *disable* to describe making a feature or command unavailable. For example:
```

- Clear the checkbox to disable automatic monitoring.
- The feature is disabled by default.

> Note that other alternative terms to *disable*—such as *deactivate*, *turn off*, or *stop*—are acceptable usage where appropriate and may be found in existing documentation. One important consideration when choosing terminology for these cases is that you make sure the language matches corresponding language in the user interface, if applicable.

Reviewer: ```suggestion
Note that alternatives to *disable*—such as *deactivate*, *turn off*, or *stop*—are acceptable usage where appropriate and may be found in existing documentation. In all cases, use language that corresponds to the language used in the UI, if applicable.
```

Author: Much better.

**enable**

> Use *enable* to describe making a feature or command available. Examples:

Reviewer: ```suggestion
Use *enable* to describe making a feature or command available. For example:
```

- Select the checkbox to enable automatic monitoring.
- The feature is enabled by default.

> Note that other alternative terms to *enable*—such as *activate*, *turn on*, *start*—are acceptable usage where appropriate and may be found in existing documentation. One important consideration when choosing terminology for these cases is that you make sure the language matches corresponding language in the user interface, if applicable.

Reviewer: ```suggestion
Note that alternatives to *enable*—such as *activate*, *turn on*, or *start*—are acceptable usage where appropriate and may be found in existing documentation. In all cases, use language that corresponds to the language used in the UI, if applicable.
```

Author: Much better. Thanks for the review.
